--- a/gloox-1.0.24/gloox.vcproj
+++ b/gloox-1.0.24/gloox.vcproj
@@ -17,7 +17,7 @@
 	<Configurations>
 		<Configuration
 			Name="Debug|Win32"
-			OutputDirectory="..\gloox-1.0\$(ConfigurationName)"
+			OutputDirectory="$(SolutionDir)$(ConfigurationName)"
 			IntermediateDirectory="$(ConfigurationName)"
 			ConfigurationType="2"
 			CharacterSet="2"
@@ -332,6 +332,10 @@
 				>
 			</File>
 			<File
+				RelativePath="src\iodata.cpp"
+				>
+			</File>
+			<File
 				RelativePath="src\iq.cpp"
 				>
 			</File>
@@ -834,6 +838,10 @@
 				>
 			</File>
 			<File
+				RelativePath="src\iodata.h"
+				>
+			</File>
+			<File
 				RelativePath="src\iq.h"
 				>
 			</File>
